showkase-processor/src/main/java/com/airbnb/android/showkase/processor/ShowkaseProcessor.kt

```diff
@@ -82,7 +102,8 @@ class ShowkaseProcessor @JvmOverloads constructor(
     private fun processComponentAnnotation(roundEnvironment: XRoundEnv): Set<ShowkaseMetadata.Component> {
         val showkaseComposablesMetadata = processShowkaseAnnotation(roundEnvironment)
         val previewComposablesMetadata = processPreviewAnnotation(roundEnvironment)
-        return (showkaseComposablesMetadata + previewComposablesMetadata)
+        val customPreviewFromClassPathMetadata = processCustomAnnotationFromClasspath(roundEnvironment)
```

Comment:
why do we look at the classpath already? we want to pick up from classpath when we want the metadata generated from the other module. At this point of time, we want to process just the current module. Is this just a naming thing?

Reply:
shouldn't you call `processCustomAnnotation` here instead? 🤔

Reply:
Yeah you are correct. Updated this in [d40c9d5](https://github.com/airbnb/Showkase/pull/303/commits/d40c9d5294899f92367fe0de61095c5336fecad4) :) Thanks for pointing it out! :)

Reply:
When thinking about this, why can't we get annotations from class path at this point? What this method does it that it checks if there is any files that has a "saved" annotation with the `ShowkaseMultiPreviewCodegenMetadata` annotation so that we in this round can check for that annotation. The `processCustomAnnotation` will run when the preview check has found an element that is itself an annotation.

Reply:
I don't quite get this. I think we have discussed this before @vinaygaba 🤔 Shouldn't we look at the classpath annotation to check the current module if we have used that annotation in the current module? Maybe I'm mistaken here.

Reply:
Yeah this comment was assuming that we aren't trying to "infer" the annotations but the users are passing it manually. Since we are going the inference route, it can be reasonable to check for the classpath here

Reply:
Aha, okay. Then I will keep it as this then :)